packages/core/src/routes/hook.ts

```diff
@@ -1,8 +1,11 @@
-import { Hooks } from '@logto/schemas';
+import { Hooks, createHookGuard, updateHookGuard } from '@logto/schemas';
 import { generateStandardId } from '@logto/shared';
-import { z } from 'zod';
+import { conditional } from '@silverhand/essentials';
+import { object, string } from 'zod';
```

Comment:
would prefer `{ z }` since it's our convention

Reply:
updated.